backend/src/services/model.ts

```diff
@@ -319,6 +322,18 @@ export async function updateModel(user: UserInterface, modelId: string, modelDif
   if (modelDiff.settings?.mirror?.destinationModelId && modelDiff.settings?.mirror?.sourceModelId) {
     throw BadReq('You cannot select both mirror settings simultaneously.')
   }
+  if (modelDiff.collaborators) {
+    const invalidUsers = []
+    modelDiff.collaborators.forEach(async (collaborator) => {
+      const userInformation = await authentication.getUserInformation(collaborator.entity)
+      if (!userInformation) {
+        invalidUsers.push(userInformation)
+      }
+    })
+    if (invalidUsers.length > 0) {
+      throw BadReq('One or more user is invalid')
```

Comment:
We could tell the user which users are invalid by doing something like:
```
// We probably need to have the same entry.kind formatting logic here to make this a nice error.
// We'll also need to replicate the plural functionality in the backend to allow this to work.
throw BadReq(
  `Failed to update ${`model.kind`}. Invalid ${invalidEntities.length === 1 ? 'entity' : 'entities'}: `${invalidEntities.join(', ')}`
  )
```

Reply:
I'd prefer to keep the message simple and add this information into a context object as we do with other errors across the app.